pkg/client/clientset/versioned/fake/clientset_generated.go

```diff
@@ -75,8 +75,3 @@ var _ clientset.Interface = &Clientset{}
 func (c *Clientset) SubmarinerV1() submarinerv1.SubmarinerV1Interface {
 	return &fakesubmarinerv1.FakeSubmarinerV1{Fake: &c.Fake}
 }
-
```

Comment:
This shouldn’t be part of this patch either.

Reply:
Sorry for the noise, this should not be here.